src/redist/targets/GenerateBundledVersions.targets

```diff
@@ -262,7 +264,7 @@
 
       <AspNetCore31RuntimePackRids Include="@(AspNetCore30RuntimePackRids)" />
       <AspNetCore50RuntimePackRids Include="@(AspNetCore31RuntimePackRids);linux-musl-arm;win-arm64" />
-      <AspNetCoreRuntimePackRids Include="@(AspNetCore50RuntimePackRids);osx-arm64;linux-s390x" />
+      <AspNetCoreRuntimePackRids Include="@(AspNetCore50RuntimePackRids);osx-arm64;linux-s390x;linux-ppc64le" />
```

Comment:
This variable is also used to define `KnownFrameworkReference` for `Microsoft.AspNetCore.App` TFM `net6.0`. Does it need to be split off into `AspNetCore60RuntimePackRids` and `AspNetCore70RuntimePackRids`?

Reply:
@omajid Since s390x was added for AspNetCoreRuntimePackRids, I have added ppc64le referring the same. Please suggest if needs to split the variable to AspNetCore60RuntimePackRids and AspNetCore70RuntimePackRids